eiffel-vocabulary/EiffelActivityTriggeredEvent.md

```diff
@@ -73,7 +73,15 @@ __Required:__ No
 __Legal targets:__ [EiffelActivityTriggeredEvent](../eiffel-vocabulary/EiffelActivityTriggeredEvent.md),
 [EiffelTestSuiteStartedEvent](../eiffel-vocabulary/EiffelTestSuiteStartedEvent.md)  
 __Multiple allowed:__ No  
-__Description:__ Identifies the activity or test suite of which this event constitutes a part.
+__Description:__ Identifies the [activity](https://github.com/eiffel-community/eiffel/blob/master/eiffel-syntax-and-usage/glossary.md#activity) or test suite of which this event constitutes a part.
+
+### PRECURSOR
+__Required:__ No  
+__Legal targets:__ [EiffelActivityTriggeredEvent](../eiffel-vocabulary/EiffelActivityTriggeredEvent.md)  
+__Multiple allowed:__ Yes  
+__Description:__ Used to declare temporal relationships between [activities](https://github.com/eiffel-community/eiffel/blob/master/eiffel-syntax-and-usage/glossary.md#activity) in a [pipeline](https://github.com/eiffel-community/eiffel/blob/master/eiffel-syntax-and-usage/glossary.md#pipeline), i.e. what other activity/activities preceded this activity. This link type applies primarily to non event-triggered activities.
+
+For more information on the usage of this link type please see [Activity Linking](../eiffel-syntax-and-usage/activity-linking.md)
```

Comment:
Please fold this sentence into the Description above. The upcoming change to a YAML-based event definition assumes that everything about a link type is part of the key/value list above.

Reply:
Sure. Will do.